src/glissade/preprocessing.py

```
@@ -77,10 +91,18 @@ def align_to_reference(results_df : pd.DataFrame, reference_file : str, database
           else:
               all_prots_string += '$'
   in_tide = [x < database_fdr_threshold for x in results_df['q-value']]
+  n_in_tide = sum(in_tide)
+  print(f"  FDR threshold: {database_fdr_threshold}")
+  n_total = len(results_df)
+  print(f"  Percolator PSMs passing FDR: {n_in_tide} / {n_total} ({100*n_in_tide/n_total:.1f}%)")
 
   db_peps = [''.join([i for i in re.sub(r'\[.*?\]', '', x[2:-2]) if i.isalpha()]).replace('I','L') for x in results_df['peptide']]
   denovo_peps = [''.join([i for i in re.sub(r'\[.*?\]', '', x) if i.isalpha()]).replace('I','L') for x in results_df['denovo_peptide']]
   agrees = [x == y for x,y in zip(db_peps, denovo_peps)]
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)